src/helpers.js

```diff
@@ -19,19 +26,41 @@ export const createLink = (href, text) => (
   </a>
 );
 
-export const createEligibilityTooltip = (eligibility) => {
+export const populateEligibilityColumn = (eligibility) => {
   return eligibility.tooltip ? (
-    <Tooltip content={eligibility.tooltip}>
-      <span>{eligibility.title}</span>
-    </Tooltip>
+    <Popover
+      distance={0}
+      position="right"
+      triggerAction="hover"
+      headerContent={
+        <Title headingLevel="h4">
+          <BanIcon className={xsIcon} /> System is not eligible to run this task
```

Comment:
Icons must be wrapped to additional `Icon` component according to PF5 docs: https://www.patternfly.org/components/icon. You can also specify the size of the icon by using `size` prop.

Reply:
Done